api/clientset/v1beta1/topology_test.go

```diff
@@ -111,7 +132,7 @@ func TestCreate(t *testing.T) {
 		}
 		t.Run(tt.desc, func(t *testing.T) {
 			tc := cs.Topology("foo")
-			got, err := tc.Create(context.Background(), tt.want)
+			got, err := tc.Create(context.Background(), tt.want, metav1.CreateOptions{})
```

Comment:
Would a test case that sets one of these options be valuable here?

Reply:
nope you never set these in any general case